src/controllers/auth.controller.js

```diff
@@ -0,0 +1,46 @@
+
+const jwt = require('jsonwebtoken');
+
+const UserModel = require('../models/user.model');
```

Comment:
Use ES6 constructs. `import` and `export`.

Reply:
In that case, are we supposed to use the babel to start the application?
Or compile the server side as well?